source/containers.tex

```diff
@@ -9520,10 +9520,8 @@
 for operations where there is additional semantic information.
 
 \pnum
-For any \tcode{N},
-\tcode{inplace_vector<T, N>::iterator} and
-\tcode{inplace_vector<T, N>::const_iterator}
-meet the constexpr iterator requirements.
+The types \tcode{iterator} and \tcode{const_iterator} meet
+the constexpr iterator requirements\iref{iterator.requirements.general}.
```

Comment:
IMO it would be better to move this paragraph after the synopsis. It seems less clear to remove `inplace_vector<T, N>::` here.